content/docs/command-reference/get.md

```diff
@@ -44,6 +44,10 @@ that DVC-tracked targets should be found in a
 [`.dvc`](/doc/user-guide/dvc-files-and-directories#dvc-files) file of the
 project.
 
+> Note that `dvc list` supports `path` granularity for directories nested inside
```

Comment:
should be natural part of the `path` description to my mind

Reply:
Meaning not a separate block quote? I wouldn't combine the paragraphs though, the previous one is already kind of long. Let me see what I can do... ⏳

Reply:
it's not about `dvc list` btw ... 

I don't know - in this specific case this note seems excessive. Not sure about other places. At the very most I would include `(including files within DVC-tracked directories)` somewhere ... and that should be enough

Reply:
Or would re-structure the paragraph a bit.

Reply:
This was done per a previous comment.